src/components/vue-form-builder.vue

```diff
@@ -762,31 +644,48 @@ export default {
           .includes(this.filterQuery.toLowerCase());
       });
 
-      const included = filtered.filter((control) => {
-        return !excludedLabels.includes(control.label);
-      });
-
-      const prioritySorted = included.sort((a, b) => {
-        const indexA = priorityLabels.indexOf(a.label);
-        const indexB = priorityLabels.indexOf(b.label);
-
-        if (indexA !== -1 && indexB !== -1) {
-          return indexA - indexB;
-        }
-        if (indexA !== -1) {
-          return -1;
+      return filtered;
+    },
+    filteredControlsGrouped() {
+      const grouped = this.filteredControls.reduce((groups, control) => {
+        let groupName = _.get(control, 'group', null);
+        
+        if (!groupName) {
+          groupName = "Advanced";
```

Comment:
Since "Advanced" is an special group and it is used in different places, consider to move to a constant